ldi-orchestrator/ldio-application/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldio/modules/DummyIn.java

```diff
@@ -3,17 +3,28 @@
 import be.vlaanderen.informatievlaanderen.ldes.ldi.services.ComponentExecutor;
 import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
 import be.vlaanderen.informatievlaanderen.ldes.ldio.types.LdioInput;
+import org.springframework.context.ApplicationEventPublisher;
 
 public class DummyIn extends LdioInput {
 	private int counter = 0;
 
-	public DummyIn(ComponentExecutor executor, LdiAdapter adapter) {
-		super("DummyIn", "test", executor, adapter, null);
+	public DummyIn(ComponentExecutor executor, LdiAdapter adapter, ApplicationEventPublisher applicationEventPublisher) {
+		super("DummyIn", "test", executor, adapter, null, applicationEventPublisher);
 	}
 
 	public void sendData() {
 		String quad = "_:b0 <http://schema.org/integer> \"" + counter++
 		              + "\"^^<http://www.w3.org/2001/XMLSchema#integer> .";
 		processInput(quad, "application/n-quads");
 	}
+
+	@Override
+	protected void resume() {
+
+	}
+
+	@Override
+	protected void pause() {
+
+	}
```

Comment:
These implementations should not be empty but entail the pausing/resume logic.

although dummys, they should make it easy to quickly test your main logic..